Ansible/ansible_collections/jfrog/platform/roles/postgres/templates/postgresql.conf.j2

```diff
@@ -97,7 +97,11 @@ max_connections = {{ postgres_server_max_connections }}			# (change requires res
 #ssl_key_file = 'server.key'		# (change requires restart)
 #ssl_ca_file = ''			# (change requires restart)
 #ssl_crl_file = ''			# (change requires restart)
+{% if fips_enabled %}
+password_encryption = scram-sha-256
+{% else %}
 #password_encryption = on
```

Comment:
probably should be `password_encryption = password`

Reply:
Setting the encryption parameter to password would turn off all encryption. When considering the available options, I didn't see any benefits of using password over scram? Could you elaborate?

Reply:
We probably should use default value `md5` instead of `password`

Reply:
If left hashed out, the default hashing is MD5. You could explicitly specify that if you feel like that's the best way to handle this?

Reply:
I would recommend to explicitly compare with 0/1 instead of casting to bool